src/config/subgraph.ts

```diff
@@ -0,0 +1,11 @@
+import { GraphQLClient } from 'graphql-request';
+import { ChainEnum } from './chains';
+
+export const subgraphClients: Partial<Record<ChainEnum, GraphQLClient>> = {
+  [ChainEnum.RSK_TESTNET]: new GraphQLClient(
+    'http://3.72.58.184:8000/subgraphs/name/babelfish/rskTestnet-graph'
+  ),
+  [ChainEnum.RSK]: new GraphQLClient(
+    'http://3.72.58.184:7000/subgraphs/name/babelfish/rsk-graph'
```

Comment:
Here use something like:
window.location + port + subgraphPath
By default the subgraph and front will be on the same machine. It will handle more cases.

Reply:
The problem is that this way it wont work on our local environment

Reply:
Thats true. Can be left as it is.